### PR TITLE
Added portuguese Thunderbird

### PR DESCRIPTION
**Pull request by alexandregz**

### CODE REVIEW

File: Casks/thunderbird-pt.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'thunderbird-pt' do
+  version '31.4.0'
+  sha256 '36fe75ceae334fabbff74ed49a30755b87a9e2f1ec6af54551794151bd74021f'
+
+  url "https://download.mozilla.org/?product=thunderbird-#{version}&os=osx&lang=pt-BR"
+  homepage 'https://www.mozilla.org/pt/thunderbird/'
+  license :oss
```
Comment (vitorgalvao) [inline Casks/thunderbird-pt.rb]: Should be `: mpl`.

**Comment by vitorgalvao** [comment]
Please add `name` and `vendor`. Follow [the main cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/firefox.rb)

**Comment by alexandregz** [comment]
[Thunderbird formula](https://github.com/caskroom/homebrew-cask/blob/master/Casks/thunderbird.rb) doesn't have name/vendor

**Comment by vitorgalvao** [comment]
You’re correct; I linked to firefox. Fixed now. Thank you.

**Comment by alexandregz** [comment]
You're right too, Thunderbird have name and vendor tags (I didn't see)

**Comment by vitorgalvao** [comment]
No, you were right. I’ve added `name` and `vendor` after your comment.

**Comment by alexandregz** [comment]
rebased too

**Comment by vitorgalvao** [comment]
Same here, and with the others. Do you want me to do it?

**Comment by alexandregz** [comment]
done

**Pull request merged**